### ISSUE TITLE
Create bonus food items

### ISSUE THREAD

**Issue by jajodiaraghav** [jajodiaraghav/Snake#44]
Bonus food items randomly appears between the game and provides bonus points if eaten by snake.

**Comment by CaptainDaVinci** [comment]
I would like to work on this.

**Comment by CaptainDaVinci** [comment]
I was thinking, maybe we could have "powerups" which have different effects (slow, fast, self-destruct, score multiplier, etc).

**Comment by jajodiaraghav** [comment]
@CaptainDaVinci You can open a new issue and we can implement that as well. Maybe something appearing less frequently that bonus food. What do you think?

**Comment by CaptainDaVinci** [comment]
Yes, we could have different frequency for different powerups, these would appear independent of the food and stay only for a limited time.
